src/TestSuite/Fixture/TestFixture.php

```diff
@@ -196,17 +197,18 @@ protected function _schemaFromImport()
         if (!is_array($this->import)) {
             return;
         }
-        $import = array_merge(
-            ['connection' => 'default', 'table' => null],
-            $this->import
-        );
+        $import = $this->import + ['connection' => 'default', 'table' => null, 'model' => null];
+
+        if (!empty($import['model'])) {
+            $import['table'] = TableRegistry::get($import['model'])->table();
```

Comment:
If both are defined should we raise an error instead? It could be unexpected if we prefer one over the other.